packages/schematics/migrations/update-8-0-0/update-8-0-0.ts

```diff
@@ -314,6 +322,7 @@ export default function(): Rule {
     updateTslintRules,
     addDependencies(),
     updateDefaultCollection,
-    formatFiles()
+    formatFiles(),
+    setRootDir
```

Comment:
We should probably do this before `formatFiles` otherwise, the file won't be formatted

Reply:
Fixed